magellan-journey-sample/src/main/java/com/wealthfront/magellan/sample/menu/MenuProvider.kt

```diff
@@ -0,0 +1,33 @@
+package com.wealthfront.magellan.sample.menu
+
+import android.view.Menu
+import android.view.MenuItem
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class MenuProvider @Inject constructor() {
+
+  private var menu: Menu? = null
+
+  fun setMenu(menu: Menu?) {
+    this.menu = menu
+    updateMenu()
+  }
+
+  fun clearMenu() {
+    menu = null
+  }
+
+  fun findItem(item: Int): MenuItem {
+    return menu!!.findItem(item)
+  }
+
+  fun updateMenu() {
```

Comment:
nit: we should make this name more descriptive, like `clearMenu` or `makeAllItemsInvisible`

Reply:
done